students/mariusz_michalczyk/lesson_06_dicts/uppercase.py

```diff
@@ -0,0 +1,20 @@
+def capitalize(lower_case_word):
+    map = {}
+    index = 0
+    for char in lower_case_word:
+        map[index] = char
+        index += 1
+    map[0] = map[0].upper()
+    return "".join(map.values())
+
+
+def capitalize_strs(strs):
```

Comment:
This can be replaced by `map(capitalize, strs)`, it's a very generic concept to apply function on each element of a sequence.